src/prommis/HD_CE.py

```diff
@@ -0,0 +1,344 @@
+from pyomo.environ import ConcreteModel, Constraint, Param, Var, units,PositiveReals, log
+from pyomo.environ import units as pyunits
+from idaes.core import FlowsheetBlock,UnitModelCostingBlock,UnitModelBlock
+from idaes.models.costing.SSLW import SSLWCosting, SSLWCostingData, VesselMaterial
+from idaes.core.solvers import get_solver
+from math import pi 
+
+
+class HDFurnaceCostEstimator:
+
+    """Cost Estimation of an Hydrogen Decrepitation Furnace"""
+
+    def __init__(self,
+                ramp_up_time = 20,                                                  #Ramp Up time (in minutes)   
```

Comment:
Standardize comment formatting - some are missing a space after the `#` . also, to fit our standard line length limitations, I suggest only using a double space before the in-line comment.

I will also note you are presuming units here which will limit generality later.